forte/base_classes/active_space_solver.h

```diff
@@ -199,6 +205,10 @@ class ActiveSpaceSolver {
     /// Read wave function from disk as initial guess
     bool read_initial_guess_;
 
+    /// Unitary matrix for orbital rotations
```

Comment:
Please make it completely clear what's the role of these matrices and where they are used. For example (and I am not sure this is even correct)
```suggestion
    /// Unitary matrices that account for orbital rotations due to semi-canonicalization
    /// These are used in ... but only for ...
```